scripts/ask_adult.sh

```diff
@@ -0,0 +1,22 @@
+# код языка
+#!/bin/bash 
+# вывод в командную строку сообщения
+echo -n "Введите значение переменной a  " 
```

Comment:
No need `-n` in a terminal
![image](https://user-images.githubusercontent.com/10350366/107885224-12298880-6f0a-11eb-8736-70fd7cd2c89d.png)